py4web/core.py

```diff
@@ -272,6 +272,7 @@ def on_success(self):
 class DAL(pydal.DAL, Fixture):
     def on_request(self):
         threadsafevariable.ThreadSafeVariable.restore(ICECUBE)
+        self._aliased_tables.clear() # should we do tha same on_error/on_success?
```

Comment:
Damn if we need to clear it, _aliased_tables is not thread safe. Need to check before merging this.

Reply:
Oh yeah, my bad, is it fixed now? (with [below commit](https://github.com/web2py/py4web/pull/207/commits/d82afd1df460cf0650bda88c46af343e633abfbb))

Reply:
testing a fix